draft-packages/menu/docs/components/MenuContentExample.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react"
```

Comment:
Nice refactoring to move this out.

The one thing I don't like about this though (and I guess that was how it was before, too) is that it makes the code of the stories "un-copy & paste-able": You can't just look at the story, click the "Code view" button and then copy the code to your code base because you have to find `<MenuContentExample />` first - which you'd have to have the Kaizen repo for.

I wish we could `spread/unwrap` the component somehow.

(Also nothing you need to solve in this PR IMO)

Reply:
Oh yeah, this isn't great from that perspective, kinda hate how storybook's code export only shows a shallow render of components like this, but I opted to go this route instead of producing a lot of duplicate code.

Originally this was in the actual stories as <MenuInstance />, which wasn't really clear that it's not an actual component, so hopefully the name will help clear up that confusion. I've also tried to clamp it down to use in examples where the inner content isn't as relevant to understanding the story. 🤞 that doesn't get lost